chaoscenter/authentication/pkg/entities/session.go

```diff
@@ -6,3 +6,24 @@ type RevokedToken struct {
 	ExpiresAt int64  `bson:"expires_at"`
 	CreatedAt int64  `bson:"created_at"`
 }
+
+// ApiTokenInput struct for storing ApiTokenInput
+type ApiTokenInput struct {
+	UserID               string `json:"user_id"`
+	Name                 string `json:"name"`
+	ExpirationDateInDays int    `json:"expiration_date_in_days"`
```

Comment:
does it store number of days remaining for its expiration?

Reply:
Yes, and I rename `expiration_date_in_days` to `days_until_expiration` to clarify.